### ISSUE TITLE
PortAudio error on Mac

### ISSUE THREAD

**Issue by lucretiomsp** [lucretiomsp/phausto#42]
on MacOsX, this OrtAudio randomly appear:
```
||PaMacCore (AUHAL)|| Error on line 2523: err='-50', msg=Unknown Error
```
the solution would be to recompile the Mac dynamic library using CoreAudio.

**Comment by sletz** [comment]
Be sure to check first that you have the latest PortAudio version,  https://files.portaudio.com/download.html ?

**Comment by lucretiomsp** [comment]
@sletz , yes I do :(. by the way, i already had on my mind to use CoreAudio on Mac

**Comment by lucretiomsp** [comment]
fixed using CoreAudio on Mac
